src/wasm/wasm-s-parser.cpp

```diff
@@ -3080,8 +3080,9 @@ Expression*
 SExpressionWasmBuilder::makeStringNew(Element& s, StringNewOp op, bool try_) {
```

Comment:
It seems `try_` operand is no longer needed in `makeStringNew` signature

Reply:
It is used for these opcodes:
https://github.com/WebAssembly/binaryen/blob/afca5f51a46750927ac9263297d24b224915e558/scripts/gen-s-parser.py#L605-L606